app/parsers/bulkrax/parser_export_record_set.rb

```diff
@@ -208,7 +208,7 @@ def collections_query
 
     class LocalCollection < Base
       def works_query
-        "local_collection_name_tesim:*#{local_coll_id} #{extra_filters}"
+        "local_collection_name_tesim:#{local_coll_id} #{extra_filters}"
```

Comment:
the `_tesim` field might be too permissive and match on tokenized splits of the URI suffixes (collection IDs) so I think we should stick with `_sim` for an exact match